src/utils/dateFormatter.ts

```diff
@@ -18,3 +20,14 @@ export const getShortTime = (dateTime: string, locale: string | undefined | null
   new Intl.DateTimeFormat(locale || 'en-US', { timeStyle: 'short', timeZone: timeZone ?? 'UTC' })
     .format(new Date(dateTime))
     .toLowerCase();
+
+// return string version for a month from number, eg. 1 -> January
+export const getMonth = (month?: number): string => {
+  const monthNumber = month?.toLocaleString('en-US', {
+    minimumIntegerDigits: 2,
+    useGrouping: false,
+  });
+
+  const stringsMap = strings as any;
+  return stringsMap[`MONTH_${monthNumber}`] ?? '';
```

Comment:
i can also use `Intl` to generate the month. Using strings since we show same info on the reports page. open to suggestions

Reply:
Just started looking at this, but the route you went with, `Intl`, seems like it should be just fine

Reply:
Thanks.. yes I felt uneasy with previous approach. Was hoping to leverage the strings but it felt forced.